archeo/inc/patterns/simple-list-of-posts.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Simple list of posts
+ */
+return array(
+	'title'      => __( 'Simple list of posts', 'archeo' ),
+	'categories' => array( 'pages' ),
+	'content'    => '<!-- wp:group {"align":"full"} -->
```

Comment:
Should this be alignfull?

Reply:
I went with alignfull because of how this pattern is used on the homepage:

![image](https://user-images.githubusercontent.com/1645628/153180812-3aa9f1ca-dcdb-400e-93db-dd894d7c77e4.png)

But maybe it should be alignwide?